src/write.c

```diff
@@ -1831,11 +1831,24 @@ static avifResult avifEncoderAddImageInternal(avifEncoder * encoder,
     }
 
     // -----------------------------------------------------------------------
-    // Map quality and qualityAlpha to quantizer and quantizerAlpha
+    // Map quality settings to quantizer values.
     encoder->data->quantizer = avifQualityToQuantizer(encoder->quality, encoder->minQuantizer, encoder->maxQuantizer);
-    encoder->data->quantizerAlpha = avifQualityToQuantizer(encoder->qualityAlpha, encoder->minQuantizerAlpha, encoder->maxQuantizerAlpha);
-    encoder->data->quantizerGainMap =
-        avifQualityToQuantizer(encoder->qualityGainMap, AVIF_QUANTIZER_BEST_QUALITY, AVIF_QUANTIZER_WORST_QUALITY);
+    // If alpha quality, and min and max quantizer have their default values, default to the same quality as color.
+    if (encoder->qualityAlpha == AVIF_QUALITY_DEFAULT && encoder->minQuantizerAlpha == AVIF_QUANTIZER_BEST_QUALITY &&
+        encoder->maxQuantizerAlpha == AVIF_QUANTIZER_WORST_QUALITY) {
+        encoder->data->quantizerAlpha = encoder->data->quantizer;
+        encoder->minQuantizerAlpha = encoder->minQuantizer;
+        encoder->maxQuantizerAlpha = encoder->maxQuantizer;
```

Comment:
We probably can leave `encoder->minQuantizerAlpha` and `encoder->maxQuantizerAlpha` unchanged.

Reply:
As explained above right now min/max quantizer actually do something (sometimes) and should be kept in sync I believe.

Reply:
I looked into this. We should delete these two lines (1840-1841).

We should not modify `encoder->minQuantizerAlpha` and `encoder->maxQuantizerAlpha` because they are owned by the caller/user. `encoder->data->quantizerAlpha`, on the other hand, is an internal data member that we own.